src/thorlabs_cube/driver/tcube/tpz.py

```diff
@@ -3,6 +3,8 @@
 from thorlabs_cube.driver.base import _Cube
 from thorlabs_cube.driver.message import MGMSG, Message, MsgError
 
+_CHANNEL = 0x01
```

Comment:
```suggestion
_CHANNEL: int = 0x01
```
Add type hints consistently!